specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -510,7 +510,6 @@
           "description": "Resource location"
         },
         "tags": {
-          "readOnly": true,
```

Comment:
As breakingchange tool detected:  1029 - ReadonlyPropertyChanged
This is a breaking change, can you I know why remove readonly, if only for fix Model validation, you can also remove it from payload of example